corehq/apps/app_manager/models.py

```diff
@@ -2168,6 +2168,9 @@ def uses_media(self):
     def uses_usercase(self):
         return False
 
+    def add_insert_form(self, from_module, form, index=None, with_source=False):
+        raise IncompatibleFormTypeException()
```

Comment:
Not striclty necessary but this method wasn't implemented by all the module subclasses so this is just a safety